CMakeLists.txt

```diff
@@ -209,11 +213,15 @@ elseif(UNIX)
 endif()
 
 if(USE_LAPACK)
-  add_definitions(-DMXNET_USE_LAPACK=1)
-  list(APPEND mxnet_LINKER_LIBS lapack)
+  if(BLAS STREQUAL "Open" OR BLAS STREQUAL "open")
+    add_definitions(-DMXNET_USE_LAPACK=1)
```

Comment:
DO unit tests pass with this?  test_laop() failed on mine with LAPACK and open blas.